### PR TITLE
Add Privacy Policy

### PR DESCRIPTION
**Pull request by erinosher**

### CODE REVIEW

**Review by tomMisson — DISMISSED** [review]
Awesome stuff

File: privacy.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+Or write to us at:<br />
+The Data Protection Officer<br />
+Hackathons UK<br />
+Suite 2202<br />
+109 Vernon House<br />
+Friar Lane<br />
+Nottingham<br />
+NG1 6DQ<br />
+United Kingdo
```
Comment (Fireshot415) [inline privacy.md]: Typo, @aaronosher ;)

**Review by Fireshot415 — CHANGES_REQUESTED** [review]
Needs typo fixing

File: privacy.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+ - How to contact us
+ - How to contact the appropriate authorities
+
+## What data do we collect?
+
+We collect the following data:
+
+ - Personal identification information (name, email address, phone number, social media usernames)
+ - What age group (13-17, or 18+) you fall into
+ - What you academic institution (if applicable) you are a part of
```
Comment (robzwolf) [inline privacy.md]: Remove the first `you`

Comment (Fireshot415) [inline privacy.md]: Remove the "you"?

File: privacy.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+## How do we collect your data?
+
+You directly provide us with most of the data we collect. We collect data process data when you:
```
Comment (robzwolf) [inline privacy.md]: `collect data` or `process data` – not both

File: privacy.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+Suite 2202<br />
+109 Vernon House<br />
+Friar Lane<br />
+Nottingham<br />
+NG1 6DQ<br />
+United Kingdom
+
+## Cookies
+
+Cookies are text files placed on your computer to collect standard internet log information and visitor behavior information. When you visit our websites, we may collect information from you automatically through cookies or similar technology.
```
Comment (Fireshot415) [inline privacy.md]: Change "behavior" to "behaviour"

**Review by Fireshot415 — CHANGES_REQUESTED** [review]
@aaronosher

**Review by robzwolf — CHANGES_REQUESTED** [review]
Couple of minor typos

**Review by robzwolf — APPROVED** [review]

**Pull request merged**